templates/emails/email-downloads.php

```diff
@@ -22,21 +22,21 @@
 
 $text_align = is_rtl() ? 'right' : 'left';
 
-?><h2 class="woocommerce-order-downloads__title"><?php _e( 'Downloads', 'woocommerce' ); ?></h2>
+?><h2 class="woocommerce-order-downloads__title"><?php esc_html_e( 'Downloads', 'woocommerce' ); ?></h2>
 
 <table class="td" cellspacing="0" cellpadding="6" style="width: 100%; font-family: 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif; margin-bottom: 40px;" border="1">
 	<thead>
 		<tr>
 			<?php foreach ( $columns as $column_id => $column_name ) : ?>
-				<th class="td" scope="col" style="text-align:<?php echo $text_align; ?>;"><?php echo esc_html( $column_name ); ?></th>
+				<th class="td" scope="col" style="text-align:<?php echo esc_html( $text_align ); ?>;"><?php echo esc_html( $column_name ); ?></th>
 			<?php endforeach; ?>
 		</tr>
 	</thead>
 
 	<?php foreach ( $downloads as $download ) : ?>
 		<tr>
 			<?php foreach ( $columns as $column_id => $column_name ) : ?>
-					<td class="td" style="text-align:<?php echo $text_align; ?>;"><?php
+					<td class="td" style="text-align:<?php echo esc_html( $text_align ); ?>;"><?php
```

Comment:
`text-align:` should use `esc_attr()`, since is a attribute, `esc_html()` is not safe enough here.